unit-tests/unit-tests-post-processing.cpp

```diff
@@ -435,26 +435,12 @@ TEST_CASE("Post-Processing expected output", "[post-processing-filters]")
     REQUIRE_THROWS(is_subset(colorizer_processed_set, original));
 
     rs2::frameset to_disp_processed_set = original.apply_filter(to_disp);
-    REQUIRE(is_subset(original, to_disp_processed_set));
     if(supports_disparity)
         REQUIRE_THROWS(is_subset(to_disp_processed_set, original));
 
     rs2::frameset from_disp_processed_set = original.apply_filter(from_disp);//should bypass
     REQUIRE(is_equal(original, from_disp_processed_set));
 
-    rs2::frameset full_pipe = original.
```

Comment:
Why did you remove it ?

Reply:
I have added another test for the full pipe, I need to upload the records before pushing it